docs/releases/2.1.txt

```diff
@@ -172,6 +172,23 @@ Models
 * The new :class:`~django.db.models.functions.Replace` database function
   replaces strings in an expression.
 
+* The new :class:`~django.db.models.functions.Chr` database function returns
+  the unicode character of an integer value.
+
+* The new :class:`~django.db.models.functions.Ord` database function returns
+  the unicode code point of the first character of a string expression.
+
+* Added the :class:`~django.db.models.functions.Left` database function.  By
```

Comment:
Follow the style of the previous notes. "The new ..." Further details like "Oracle and SQLite do not have a Left function.  Django will use their Substr function in its place." should be in the function's documentation rather than the release notes.